docs/rebase.md

```diff
@@ -83,6 +95,10 @@ When running this command, you will be prompted for the branch on top of which y
 
 If any merge conflicts occur while rebasing, the state of the operation will be reflected in the dashboard, with successful rebased commits showing a check mark, and the commit with the current merge conflict showing an X as well as a list of files with conflicts.
 
+#### Toggle perserve merges mode (`m`)
+
+In default, merges will be lost when rebasing. If perserve merges mode is on, commits within a merge are consolidated in a single item. If any conflicts occur while rebasing a merge commit, the commits within the merge will be shown and the conflicting commit will be marked.
```

Comment:
**REPOST** my mobile app posted in a wrong location.

Actually, the doc of `git rebase -p` says

> This uses the --interactive machinery internally, but combining it with the --interactive option explicitly is generally not a good idea unless you know what you are doing (see BUGS below).